src/test/py/bazel/bzlmod/mod_command_test.py

```diff
@@ -1270,7 +1270,6 @@ def testModTidyWithIncludes(self):
       self.assertEqual(
           [
               'include("//:firstProd.MODULE.bazel")',
-              '',  # formatted despite no extension usages!
```

Comment:
should we introduce some other formatting error in the original file (like extra spaces between `include` and `(`) and test that it's gone after formatting?

Reply:
We have a dedicated test for this: https://github.com/bazelbuild/bazel/blob/39c2360f3b8ada263d9bbe9ddb2a469177a9bf65/src/test/py/bazel/bzlmod/mod_command_test.py#L941